.evg.yml

```diff
@@ -355,6 +355,7 @@ functions:
     command: shell.exec
     type: test
     params:
+      shell: bash
```

Comment:
Drive by fix, saw in evergreen it was complaining about the SECONDS variable which is supported in `bash` and not `sh`.